server/core/store.go

```diff
@@ -302,14 +302,23 @@ func (s *StoreInfo) regionScoreV1(highSpaceRatio, lowSpaceRatio float64, delta i
 }
 
 func (s *StoreInfo) regionScoreV2(delta int64, deviation int) float64 {
+	var amplification float64
 	A := float64(float64(s.GetAvgAvailable())-float64(deviation)*float64(s.GetAvailableDeviation())) / gb
 	C := float64(s.GetCapacity()) / gb
-	R := float64(s.GetRegionSize() + delta)
 	var (
 		K, M float64 = 1, 256 // Experience value to control the weight of the available influence on score
 		F    float64 = 20     // Experience value to prevent some nodes from running out of disk space prematurely.
 	)
+	used := float64(s.GetUsedSize()) / mb
+	if s.GetRegionSize() == 0 || used == 0 {
+		amplification = 1
+	} else {
+		// because of rocksdb compression, region size is larger than actual used size
+		amplification = float64(s.GetRegionSize()) / used
+	}
 
+	// As the delta, not directly use the used size.
+	R := float64(s.GetRegionSize()+delta) / amplification
```

Comment:
we will replace used_size with region_size?

Reply:
Yes, `region_size` may not correspond to the `used_size`, we prefer to balance `used_size`. see:
![image](https://user-images.githubusercontent.com/6428910/112784433-4b662400-9084-11eb-813c-fbec1ff56a67.png)
![image](https://user-images.githubusercontent.com/6428910/112784578-9ed87200-9084-11eb-80cf-dc05562dd9c3.png)

In the test, the region_size be more balance but  the `available_ratio` is not.

Reply:
cc @disksing